src/transformation/utils/lua-ast.ts

```diff
@@ -143,9 +141,12 @@ export function createLocalOrExportedOrGlobalDeclaration(
             );
         }
     } else {
-        const insideFunction = findScope(context, ScopeType.Function) !== undefined;
+        const isTopLevelVariable = peekScope(context).type === ScopeType.File;
```

Comment:
```suggestion
		const scope = peekScope(context);
        const isTopLevelVariable = scope.type === ScopeType.File;
```

Scope is used below anyway